internal/consensus/reactor_test.go

```diff
@@ -89,9 +92,13 @@ func setup(t *testing.T, numNodes int, states []*State, size int) *reactorTestSu
 		blocksSub, err := state.eventBus.Subscribe(context.Background(), testSubscriber, types.EventQueryNewBlock, size)
 		require.NoError(t, err)
 
+		fsSub, err := state.eventBus.Subscribe(context.Background(), testSubscriber, types.EventQueryFastSyncStatus, size)
```

Comment:
I'm not psyched about making n contexts for subscriptions that don't get cancled. Presumably there's a cancellation channel that will release the resources in between tests, but it would probably be more clear, if, there was one context  that was created on line 74, with something like: 

```
ctx, cancel := context.WithCancel(context.Background))
t.Cleanup(func() { cancel() })
```